config/src/lib.rs

```diff
@@ -153,6 +153,27 @@ struct Cli {
     /// should manage
     #[clap(short, long, value_parser)]
     pub wallet_file: Option<String>,
+
+    // -------------
+    // | Telemetry |
+    // -------------
+
+    /// Whether or not to enable OTLP tracing
+    #[clap(long, value_parser)]
+    pub otlp_enabled: bool,
```

Comment:
do we need this, can one of the others below be used to determine whether OTLP is enabled? The fewer flags the better

Reply:
Also can we set `long="otlp"` here to give the switch `--otlp`

Reply:
i was considering this, but i think it's a better experience to 1) have a flag that clearly defines its intent, and 2) follow conventions in the expected manner, in this case by using the same default collector endpoint as is expected in OTLP generally (which also means the flag can be omitted if you're using defaults)

But ack on changing the `long` name